### PR TITLE
Fix failing integration test (signup spec)

### LINKED ISSUE #852: Fix failing integration test
Fix the following two integration tests in [signup.spec.js](https://github.com/deep-learning-indaba/Baobab/blob/develop/integration-tests/cypress/integration/signup.spec.js) - it seems to be an issue with the test because the actual signup page is working fine. 
- Signup form cannot be submitted in if you havent accepted the policy or if you are underage.
- Singup form can be submitted if you fill in everything correctly.

### PR DESCRIPTION
**Pull request by gausam**
It created an actual user account in the test, which on the next run would of course be unavailable for re-registration with the same credentials.

Closes #852

### CODE REVIEW

File: integration-tests/cypress/integration/signup.spec.js [hunk 1]
```diff
@@ -68,10 +69,12 @@ describe("Sign up", function() {
     cy.get("#btn-signup-confirm").should("be.disabled");
   });
 
-  it("Singup form can be submitted if you fill in everything correctly.", function() {
+  it("Signup form can be submitted if you fill in everything correctly.", function() {
     let user = testUser();
-    cy.visit("/createAccount");
```
Comment (gausam) [inline integration-tests/cypress/integration/signup.spec.js:73]: Router behavior will redirect away from `/createAccount` if the user is not authenticated.

**Comment by gausam** [comment]
> Timed out retrying: Expected to find element: '#title', but never found it.
Hmm. Seems it doesn't run fast enough on CircleCI 🤔

File: integration-tests/cypress/integration/signup.spec.js (hunk 1 above)
Comment (avishkar58) [reply integration-tests/cypress/integration/signup.spec.js:73]: Really? That seems like a bug then... /createAccount should be accessible if the user is not authenticated...

Comment (gausam) [reply integration-tests/cypress/integration/signup.spec.js:73]: Yup, that's the behavior I get on my end at least.
Should I fix the bug as well?

Comment (avishkar58) [reply integration-tests/cypress/integration/signup.spec.js:73]: Ah I get that behaviour too... yes please fix that bug as well, it makes sense for `/createAccount` to be accessible by a non-authenticated user

**Comment by avishkar58** [comment]
Hi @gausam - is this ready to be merged? It seems to be passing on CircleCI now? :D

**Comment by gausam** [comment]
@avishkar58 It's ready! 🚀

**Review by avishkar58 — APPROVED** [review]

**Pull request merged**